### ISSUE TITLE
Function to save a csv file with basic results from bigfish.stack.extract_cell

### ISSUE THREAD

**Issue by Henley13** [fish-quant/big-fish#18]
We should gather information from cell extraction in a dataframe like object and export in a a readable format.

**Comment by muellerflorian** [comment]
a csv fiel would be great. Important information it should contain is

1. Name of image.
2. Cell label.
3. Statistics of RNA counts, ideally (if present) also information about sub-cellular compartments, e.g. how many RNAs in cytoplasm vs nucleus.
4. Statistics of foci (if present), and the ideally also split it up sub-cellular compartments (if present).

**Comment by Henley13** [comment]
Issue answered in:
c97f55f7a80df84e47fd12feef9afdc0078540ed
553fadfc46ceea573d867be8775144f640e44b38

1) I did not add the name of the fov because it's an external information, we cannot find in bigfish.stack.extract_cell. This kind of metadata, like the drug used, the author or the gene coud be easily add in the dataframe before exporting it in csv. It would be the same value for all the dataframe.
2) Ok.
3) Ok.
4) I extract basic information about the cell, the nucleus and the rna. But for others detected elements (foci, translation factories, centroid), it depends what was given as input for the cell extraction. I just count their occurrences per cell. Splitting them between different sub-cellular compartments should be done before extracting results.

Basically, there are three steps:
- We get segmentation and detection results at the fov level. Potentially, we assign different detections to specific segmented areas.
- Cell extraction only consists in matching segmentation and detection results to cells. We do not analyze anything at this stage. 
- We count the number of elements per cell and summarize them in a dataframe. For more detailed statistics, we should compute them directly, outside bigifish.stack.summarize_extraction_results.
